tensorflow/python/ops/script_ops.py

```diff
@@ -546,20 +546,20 @@ def my_func(x):
     in that
     server (e.g. using `with tf.device():`).
     
-  * It produces tensors of unknown shape and rank as shape inference does not
-    work on arbitrary Python code.
-    If you need the shape you need to set it based on statically available information.
+  Note: It produces tensors of unknown shape and rank as shape inference 
+    does not work on arbitrary Python code.
+    If you need the shape, you need to set it based on statically 
+    available information.
     
     E.g.
     ```python
-    import tensorflow.compat.v2 as tf
+    import tensorflow as tf
     import numpy as np
 
-    #make magical data that requires python
-    def make_data(i): 
-        return np.cast[np.uint8](i) * np.ones([20,256,256,3], dtype=np.float32) / 10.
+    def make_synthetic_data(i):
+        return np.cast[np.uint8](i) * np.ones([20,256,256,3], 
+                dtype=np.float32) / 10.
 
-    #clean up magical data
     def make_clean_data(i): 
```

Comment:
minor: I would prefer to rename `make_clean_data` with `process_fn`

Reply:
You are too punctilious towards yourself in the past :stuck_out_tongue_winking_eye: